otx/algorithms/classification/configs/base/data/data_pipeline.py

```diff
@@ -17,29 +17,35 @@
 # pylint: disable=invalid-name
 
 __img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
-__img_size = 224
+__resize_target_size = 224
 
-train_pipeline = [
-    dict(type="Resize", size=__img_size),
+__train_pipeline = [
+    dict(type="Resize", size=__resize_target_size),
     dict(type="RandomFlip", flip_prob=0.5, direction="horizontal"),
     dict(type="AugMixAugment", config_str="augmix-m5-w3"),
     dict(type="RandomRotate", p=0.35, angle=(-10, 10)),
-    dict(type="ToNumpy"),
+    dict(type="PILImageToNDArray", keys=["img"]),
```

Comment:
This pipeline is known as the supervised learning pipeline. Any reason to change?

Reply:
To align with semi-sl pipeline. I think we don't need to maintain `ToNumpy`.

Reply:
I don't understand. As far as I know, `ToNumpy` is a pipeline provided by `mmcls`. Do we need to modify this if nothing else than `PILImageToNDArray`? Any difference?

Reply:
`PILImageToNDArray` do same work with `ToNumpy`, but it supports multiple key inputs.

Reply:
So this is like `ToNumpy` but for multiple image keys supporting (for `semisl`)?

Reply:
Yes, right